metagraph/src/annotation/row_diff_builder.cpp

```diff
@@ -243,13 +403,13 @@ void convert_batch_to_row_diff(const std::string &graph_fname,
     });
 
     // free memory occupied by sources
-    std::vector<std::unique_ptr<ColumnCompressed<>>>().swap(sources);
+    sources.clear();
```

Comment:
I don't think this guarantees that memory will be freed - only swapping with an empty vector, or calling `shrink_to_fit` afterwards does.

Reply:
Yeah, the internal array may not be released, but all objects will be destroyed, that's enough here. As a residual, we'll have just a buffer for pointers